index.md

```diff
@@ -617,6 +621,69 @@ export default {
 
 Now that we have defined a new component called `SchemaFormWithPlugins`, you can use it as you normally use any other component in your application.
 
+#### Using locally imported components
+
+The second parameter accepted by the `SchemaFormFactory` function is an `Object` with a key-value pair for component registration. This object will get merged into the `components: {}` declaration for generated `SchemaForm` component.
+
+This option will come in useful in cases where you do not want to import your form components globally, and need to declare them locally in the component that is instantiating your form.
+
+In the following example, two components `FormSelect` and `FormText` are imported locally into the file. They are injected into the `SchemaFormFactory` as the second parameter (the plugin array remains empty since no plugins are being used).
+
+```html
+<template>
+  <div id="app">
+    <SchemaForm :schema="schema" v-model="form" />
+  </div>
+</template>
+
+<script>
+import { ref, markRaw } from "vue"
+import { SchemaFormFactory } from "formvuelate"
+import FormText from "@/components/FormText"
+import FormSelect from "@/components/FormSelect"
+
+markRaw(FormSelect)
+markRaw(FormText)
+
+// Declare FormText and FormSelect as local components
+const factory = SchemaFormFactory([], { FormText, FormSelect })
+
+export default {
+  name: "App",
+  components: { SchemaForm: factory },
+  setup () {
+    const form = ref({
+      name: '',
+      pet: 'cat'
+    })
+
+    // We can now declare our `component` property as a string, since
+    // the component will be registered locally within the SchemaForm component
+    const schema = ref({
+      name: {
+        component: 'FormText',
+        label: 'Your name'
+      },
+      pet: {
+        component: 'FormSelect',
+        label: 'Your pet',
+        options: ['cat', 'dog']
+      }
+    })
+
+    return {
+      form,
+      schema,
+    }
+  },
+}
+</script>
+```
+
+Notice that the `schema` declares the `component` property now as a `String`. This is because the generated `SchemaForm` can now register both the injected components locally.
```

Comment:
```suggestion
Notice that the `schema` declares the `component` property as a `String`. This is because the generated `SchemaForm` can now register both the injected components locally.
```